tests/dummy/app/templates/components/collapsible-group.hbs

```diff
@@ -1,6 +1,6 @@
 <li class="ember-power-select-group" aria-disabled={{ember-power-select-true-string-if-present disabled}} role="option">
   <span class="ember-power-select-group-name collapsible {{if collapsed 'is-collapsed'}}" onclick={{toggle 'collapsed' this}}>{{groupName}}</span>
-  {{#unless collapsed}}
+  <div class="{{if collapsed 'hide-options'}}">
```

Comment:
I do too.
We were get exceptions though when using the up and down arrow keys, which now that I think about it highlights how the example isn't accessible. Investigating.